astroquery/esa/jwst/core.py

```diff
@@ -39,6 +39,21 @@
 __all__ = ['Jwst', 'JwstClass']
 
 
+def _check_rename_to_gz(filename):
```

Comment:
I wonder whether it would make sense to have some esa utilities somewhere that lets some shared functions to be reused rather than repeated? (certainly not required for this PR, I'm just wondering about it). It could live either under `esa/utils` or `utils/esa`. I suppose this, and some of the test mock classes can be dumped in there for sure, but maybe there is more.

Reply:
I have removed this from ESA HST and JWST modules and created a new one under esa/utils. It is included in my latest commits.